extract_endpoint/tests/test_post_to_endpoint.py

```diff
@@ -90,9 +96,13 @@ def test_post_stream(azure_service: blob.BlockBlobService,
                      upload_url: str,
                      sample_stream: NamedStream,
                      sample_stream_content: str,
-                     sample_filename: str) -> None:
+                     sample_filename: str,
+                     sample_timestamp: str) -> None:
 
-    post_return = post_to_endpoint.post_stream(stream=sample_stream, filename=sample_filename, url=upload_url)
+    post_return = post_to_endpoint.post_stream(stream=sample_stream,
+                                               filename=sample_filename,
+                                               url=upload_url,
+                                               timestamp=sample_timestamp)
     assert post_return.status_code == HTTPStatus.CREATED
     check_file_in_azure(azure_service, azure_emulator_coords, sample_filename, sample_stream_content)
 
```

Comment:
you should also use `check_file_in_azure` to make sure the timestamp file is there. (and also below)